### PR TITLE
[13.0][MIG] account move template

### PR DESCRIPTION
**Pull request by ValentinVinagre**
Taxes adapted to new v.13 format.
Now you can chose if the line is refund or not, It's a must to map the tax repartition line properly. The new tax tags are mapped too.

### CODE REVIEW

**Comment by ValentinVinagre** [comment]
Hi,
@HaraldPanten  @alexis-via  could you review please?

Thanks a lot,

**Review by HaraldPanten — APPROVED** [review]
Functional review 👍

**Comment by desdelinux** [comment]
Functional review. Test is ok. +1

**Comment by ValentinVinagre** [comment]
Hi @crece365  thanks for the review, can you approve it? thanks

**Review by desdelinux — APPROVED** [review]
Functional review. Test is ok. +1

**Review by juliap98 — APPROVED** [review]
Funcional review ok

**Comment by ValentinVinagre** [comment]
This is ready to review/merge @OCA/accounting-maintainers

**Comment by ValentinVinagre** [comment]
@OCA/accounting-maintainers ready to review.

**Comment by alexis-via** [comment]
@ValentinVinagre I wanted to make a (small) PR to your branch, but I can't create a PR to it ; it seems you have setup some branch restrictions that prevent community members from making a PR to it. I must say that it's not the right solution for community work.
My suggestion was small, just a change to use the group analytic.group_analytic_tags for the fields analytic_tag_ids. The change is located in this commit:
https://github.com/akretion/account-financial-tools/commit/cc7ebbb841fb852d04fee2fb69dc7ee961f85b4f

I have 2 other remarks:
1) why do you inherit the field "tax_repartition_line_id" on account.move.line to set readonly=False (the account module definies it as readonly=True, cf https://github.com/odoo/odoo/blob/13.0/addons/account/models/account_move.py#L2560
2) anyway, you shouldn't inherit "account.move.line" in a python file located under the wizard subdir. You should do that under the models subdir.

File: account_move_template/wizard/account_move_template_run_view.xml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+                            <field name="partner_id" />
+                            <field
+                                name="analytic_account_id"
+                                domain="[('company_id','=',company_id)]"
+                                groups="analytic.group_analytic_accounting"
+                            />
+                            <field
+                                name="analytic_tag_ids"
+                                widget="many2many_tags"
+                                groups="analytic.group_analytic_accounting"
```
Comment (alexis-via) [inline account_move_template/wizard/account_move_template_run_view.xml]: Please use groups="analytic.group_analytic_tags" here too.

Comment (ValentinVinagre) [reply account_move_template/wizard/account_move_template_run_view.xml]: done

**Review by HaraldPanten — APPROVED** [review]

**Comment by HaraldPanten** [comment]
Hi @alexis-via could you review? Your suggestions have been applied 👍

**Comment by HaraldPanten** [comment]
Hi @alexis-via ,

Could you validate this one?

**Review by ps-tubtim — APPROVED** [review]

**Comment by ValentinVinagre** [comment]
Hi @pedrobaeza ,
I think it's ready to merge. What do u think?

**Comment by pedrobaeza** [comment]
I can merge it blindly due to approvals, but I haven't checked by myself:

/ocabot merge nobump

**Pull request merged**